### PR TITLE
Add community_member_description, tagline and mascot_image_url  to SiteConfig. Enable Cloudinary for primary_sticker_image

### PR DESCRIPTION
**Pull request by JibranKalia**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [X] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Adding `community_member_description`, `tagline` and `mascot_image_url` to SiteConfig. Also enabling cloudinary for `primary_sticker_url` which was created by #6746. 

The only thing missing in order to complete issue https://github.com/thepracticaldev/dev.to/issues/5477 is implementing `secondary_sticker_image_url`. However, that purple logo is implemented in jsx. I am not really sure how to push SiteConfig details into javascript.

## Related Tickets & Documents
#5477

### community_member_description

Screen Shot 2020-03-23 at 8 18 09 AM

Screen Shot 2020-03-23 at 8 35 07 AM

### tagline

Screen Shot 2020-03-23 at 8 31 00 AM

Screen Shot 2020-03-23 at 8 31 19 AM

### mascot_image_url

Screen Shot 2020-03-23 at 9 06 11 AM

Screen Shot 2020-03-23 at 9 06 03 AM

## Added tests?

- [X] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [X] no documentation needed

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -61,7 +61,7 @@ def icon_url(name)
   end
 
   def cloudinary(url, width = nil, _quality = 80, _format = "jpg")
-    return url if Rails.env.development? && (url.blank? || url.exclude?("http"))
+    return url if Rails.env.development? && (url.blank? || url.exclude?("http") || url.include?("fastly"))
```
Comment (JibranKalia) [inline app/helpers/application_helper.rb]: Not very happy with this new condition. However, I believe we want to keep cloudinary option for dev setup and this is the only way I could think of getting this to work.

File: app/views/layouts/_signup_modal.html.erb [hunk 2]
```diff
@@ -5,17 +5,10 @@
...
   <div class="global-signup-modal--inner-a">
-    <%= image_tag(cl_image_path(asset_path("/assets/sloan.png"),
-                                type: "fetch",
-                                width: 300,
-                                crop: "imagga_scale",
-                                quality: "auto",
-                                flags: "progressive",
-                                fetch_format: "auto",
-                                sign_url: true), class: "sloan", alt: "Sloan, the sloth mascot", loading: "lazy") %>
+    <%= image_tag(cloudinary(SiteConfig.mascot_image_url, 300), class: "mascot-image", alt: "Sloan, the sloth mascot", loading: "lazy") %>
```
Comment (JibranKalia) [inline app/views/layouts/_signup_modal.html.erb]: There are several other places in the code that calls "/assets/sloan.png" directly. Should I go ahead and change all the other places to use SiteConfig. I am leaning towards keeping this PR small.

Comment (rhymes) [reply app/views/layouts/_signup_modal.html.erb]: good call, let's do one step at a time :)

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (rhymes) [reply app/helpers/application_helper.rb]: I'm not sure about this, I understand the intention but I feel it should be part of a larger discussion and review about making cloudinary optional or not. If so, we should still probably have a separate PR that does it and a test. Can we table this for now? Maybe open an issue to submit for review

Comment (JibranKalia) [reply app/helpers/application_helper.rb]: I am completely okay with tabling it. The only downside would be that in local dev the site would look like: 

Screen Shot 2020-03-24 at 5 36 06 PM

File: app/views/layouts/_signup_modal.html.erb (hunk 2 above)
Comment (JibranKalia) [reply app/views/layouts/_signup_modal.html.erb]: Awesome.

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (rhymes) [reply app/helpers/application_helper.rb]: Got it, I asked for help for the review, let's keep it in in the meantime. Thank you!

Comment (JibranKalia) [reply app/helpers/application_helper.rb]: 👍

Comment (JibranKalia) [reply app/helpers/application_helper.rb]: Just wanted to follow up on this.

**Comment by JibranKalia** [comment]
Closing this in favor of https://github.com/thepracticaldev/dev.to/pull/7056. I messed up the git commits for this and am not sure how to recover.

**Pull request closed without merging**